src/tools/run-make-support/src/external_deps/rustdoc.rs

```diff
@@ -71,14 +71,8 @@ impl Rustdoc {
         self
     }
 
-    /// Specify path to the output folder.
-    pub fn output<P: AsRef<Path>>(&mut self, path: P) -> &mut Self {
-        self.cmd.arg("-o");
-        self.cmd.arg(path.as_ref());
-        self
-    }
-
     /// Specify output directory.
+    #[doc(alias = "output")]
```

Comment:
Remark: nice cheeky doc alias 😆

Reply:
It's super convenient. Even for me since I generally look for `output` in the docs, not `out_dir`. I'm future-proofing for myself. :laughing: